src/screens/RegisterCompleted.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {
+  SafeAreaView,
+  View,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  Image,
+} from 'react-native';
+import {Colors} from '../utils/Colors';
+
+const RegisterCompleted = ({navigation}) => {
+  return (
+    <SafeAreaView style={styles.mainContainer}>
+      <View style={styles.imageContainer}>
+        <Image
+          style={styles.heartHandsImage}
+          source={require('../assets/images/heart-handshake.png')}
+        />
+      </View>
+      <View style={styles.succesContainer}>
+        <View style={styles.mainText}>
+          <Text style={styles.title}>Tu cuenta se ha creado</Text>
+          <View style={styles.textContainer}>
+            <Text style={styles.paragraphs}>
+              Gracias por registrarte para protegerte y a proteger a tu
+              comunidad.
+            </Text>
+            <Text style={styles.paragraphs}>
+              Juntos podemos detener la propagación del COVID-19
+            </Text>
```

Comment:
Put the text in a constant message on the file `src/utils/constants` and import them